src/main/java/core/basesyntax/service/FigureSupplier.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.service;
+
+public class FigureSupplier {
```

Comment:
The `FigureSupplier` class should include a method to return a random figure. Consider implementing a method like `public Figure getRandomFigure()` that randomly selects and returns a figure with random parameters.